frontend-build-pipeline.yml

```diff
@@ -16,30 +29,30 @@ steps:
 
 - script: |
     npm ci
-    npx ng build --prod
+    npx ng build
   displayName: 'npm Build'
 
 # skipped failing unit tests
 # https://jasmine.github.io/tutorials/your_first_suite#section-Pending_Specs
 - script: |
-    npx ng test --watch=false
+    npx ng test --watch=false --browsers=ChromeHeadless
   displayName: 'Run Unit Tests'
 
 - task: SonarCloudPrepare@1
   inputs:
     SonarCloud: 'SonarCloud'
-    organization: '1054-daniel-felleman-revature-swag-shop'
+    organization: ${{ parameters.repoOrganization }}
     scannerMode: 'CLI'
     configMode: 'manual'
-    cliProjectKey: '1054-Daniel-Felleman-Revature-Swag-Shop_RevatureShopFrontend'
-    cliProjectName: 'RevatureShopFrontend'
+    cliProjectKey: ${{ parameters.sonarCloudProjectKey }}
+    cliProjectName: $ {{ parameters.sonarCloudProjectName }}
     cliSources: '.'
```

Comment:
include properties to identify the sources, coverage exclusions, and eventually the location of the coverage report file.